packages/vscode/src/plugins/prisma-postgres-manager/utils/spawnPpgDevServer.ts

```diff
@@ -0,0 +1,39 @@
+async function main() {
+  const name = process.argv[2]
+  const port = +process.argv[3]
+  const databasePort = +process.argv[4]
+  const shadowDatabasePort = +process.argv[5]
+
+  console.log(`[PPG Dev] Starting process (${process.pid}) for database: ${name}`)
+
+  if (!name || !port || !databasePort || !shadowDatabasePort) {
+    console.log('[PPG Dev] Missing argument, server cannot be started')
+    process.exit(1)
+  }
+
+  try {
+    console.log('[PPG Dev] Starting server...')
+
+    const { unstable_startServer } = await import('@prisma/dev')
+
+    await unstable_startServer({ persistenceMode: 'stateful', name, port, databasePort, shadowDatabasePort })
```

Comment:
you need to receive the server object and then `.close()` on shutdown.

```suggestion
    const server = await unstable_startServer({ persistenceMode: 'stateful', name, port, databasePort, shadowDatabasePort })
```